awx/main/tests/functional/test_rbac_job.py

```diff
@@ -368,3 +376,8 @@ def test_system_job_template_superuser(self, admin_user, system_job_template):
         access = SystemJobTemplateAccess(admin_user)
         assert access.can_read(system_job_template)
         assert access.can_start(system_job_template)
+
+    def test_org_auditor_view_system_job(self, system_job_template, org_auditor):
```

Comment:
I see you have added a test for the auditor user viewing a system job template, but I'm not finding a test for the super user viewing a system job. 

If that test already exists, why wasn't it failing and surfacing this error?
If that test doesn't already exist would you mind adding on this PR ?